configs/opencv-ovms/grpc_go/Dockerfile

```diff
@@ -27,4 +27,4 @@ COPY . .
 RUN go mod tidy
 RUN go build -o grpc-go && chmod +x entrypoint.sh
 
-ENTRYPOINT ["./entrypoint.sh"]
+#ENTRYPOINT ["./entrypoint.sh"]
```

Comment:
if this is not needed, maybe we remove it instead of commented out

Reply:
removed